extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -248,9 +247,9 @@ private function clearUsers() {
 	 * @return string homepage URL. Default is US homepage.
 	 */
 	private function getHomepageUrl() {
-		global $wgLang;
+		global $wgContLang;
 		if ( class_exists('WikiaLogoHelper') ) {
-			return ( new WikiaLogoHelper() )->getCentralUrlForLang( $wgLang->getCode() );
+			return ( new WikiaLogoHelper() )->getCentralUrlForLang( $wgContLang->getCode() );
```

Comment:
Here you use `$wgContLang->getCode()` but in `language.content` it's `$wgLanguageCode`. What's the difference? Is it the right thing to do? 

/cc @bognix @vforge